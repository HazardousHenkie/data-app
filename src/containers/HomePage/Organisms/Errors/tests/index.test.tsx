import React from 'react'
import renderer from 'react-test-renderer'

import MockingAppComponent from 'utils/testing/MockingAppComponent'

import ErrorSnackbars from '../index'

describe('<ErrorSnackbars />', () => {
    it('should render like snapshot', () => {
        const component = renderer.create(
            <MockingAppComponent>
                <ErrorSnackbars />
            </MockingAppComponent>
        )

        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})
