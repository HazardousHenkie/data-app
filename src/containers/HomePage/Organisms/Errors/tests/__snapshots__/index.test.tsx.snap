// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorSnackbars /> should render like snapshot 1`] = `null`;
